### DISCUSSION TITLE
6月个人网站-使用Wire进行Golang依赖注入 --- June Personal Web - Golang Dependency Injection Using Wire

### DISCUSSION THREAD

**Discussion by giscus** [ClavinJune/ClavinJune.github.io#24] [General]
# 6月个人网站-使用Wire进行Golang依赖注入 --- June Personal Web - Golang Dependency Injection Using Wire

Implementing dependency injection in Golang using Google Wire

https://clavinjune.dev/en/blogs/golang-dependency-injection-using-wire/

**Comment by yangtuooc** [comment]
Thank you for sharing this article about Wire dependency injection! I'm particularly interested in the project structure you've demonstrated - the domain-driven organization approach looks very clean and practical.

I'd love to learn more about the best practices and experiences when using this structure in real projects. If you have any other articles or example projects to recommend, I would be very interested in learning from them.

Thanks again for sharing!

**Reply by clavinjune** [reply]
Hi @yangtuooc  thank you for reading! sadly I don't have any publicly accessible project example. If I'm planning to have one, I'll update the blog
